recipes/msisensor-pro/build.sh

```diff
@@ -1,7 +1,37 @@
-export CPATH=${PREFIX}/include
-export INCLUDES+="-I$PREFIX/include -I$PREFIX/include/ncurses -Ivendor/samtools-0.1.19"
-export LDFLAGS="-L$PREFIX/lib -ltinfo"
-export LIBRARY_PATH=${PREFIX}/lib
-make INCLUDES="-I$PREFIX/include -I$PREFIX/include/ncurses -I$SRC_DIR/vendor/samtools-0.1.19" LIBCURSES="-L$PREFIX/lib -lncurses -ltinfo" LIBPATH="-L$PREFIX/lib" CC=$CC CXX=$CXX CFLAGS="-g -Wall -O2 -I$PREFIX/include -L$PREFIX/lib" -C$SRC_DIR/cpp/ 
+
+#$SRC_DIR/INSTALL
+#cd $SRC_DIR/cpp
 #make CC=$CC
-cp $SRC_DIR/cpp/msisensor-pro $PREFIX/bin
+
+export CPATH=${PREFIX}/include
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export CPPFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+#export CC=${PREFIX}/bin/gcc
+#export CXX=${PREFIX}/bin/g++
+echo ==============================
+#which gcc
+#which g++
+#echo ==============================
+cd ${SRC_DIR}/vendor/htslib-1.11
+autoheader
+autoconf
+./configure 
+make 
```

Comment:
htslib 1.11 is listed as a dependency. Do we need to additionally compile it here?

Reply:
Yes!

Reply:
Why?